services/app/src/components/base/Dialogs.tsx

```diff
@@ -249,7 +253,7 @@ export class TextAreaDialog<T extends BaseDialogProps> extends React.Component<T
         </DialogContent>
         <DialogActions>
           <Button id="cancelButton" onClick={() => this.props.onClose()}>Cancel</Button>
-          <Button id="submitButton" className="primary" onClick={() => this.onSubmit()}>{this.action || 'Submit'}</Button>
+          <Button id="submitButton" className="primary" onClick={() => {this.onSubmit(); this.clearText(); }}>{this.action || 'Submit'}</Button>
```

Comment:
@smartin015 figured out a way to do this more standard-ly (by having it exist at the top level TextAreaDialog), let me know if this implementation is ok